src/implementations/vanilla/components/gallery/gallery.scss

```diff
@@ -99,9 +100,18 @@ $_description-height-desktop: 108px;
   }
 }
 
-@supports (object-fit: cover) {
-  .ecl-gallery__image-container {
-    background-color: map.get($gallery, 'description-background');
+.ecl-gallery__picture {
+  display: flex;
+  height: 100%;
+
+  &:not(.ecl-gallery__slider-image) {
+    display: none;
+    flex-grow: 1;
+  }
+
+  * {
```

Comment:
why do we need this..? From what i see these two properties are overridden but this will still apply some styles to the "sources" although it doesn't seem to break anything, but still it seems this could be removed.